src/containers/tud/components/ContextualQuestionsIntro.js

```diff
@@ -3,29 +3,32 @@
 import React from 'react';
 
 import { Typography } from 'lattice-ui-kit';
+import { DateTime } from 'luxon';
 
+import TranslationKeys from '../constants/TranslationKeys';
+
+// selected activity -> map to key -> get the corresponding str
 type Props = {
   selectedActivity :string;
-  time :string;
+  time :DateTime;
+  trans :(string, ?Object) => Object;
 };
 
-const ContextualQuestionsIntro = ({ selectedActivity, time } :Props) => (
-  <>
-    <Typography variant="body2">
-      <span>Now, let&apos;s talk about what your child did while </span>
-      <span>
-        <strong>
-          { selectedActivity }
-        </strong>
-      </span>
-      <span> at </span>
-      <strong>
-        { time }
-      </strong>
-      <span>.</span>
+const ContextualQuestionsIntro = ({ selectedActivity, time, trans } :Props) => {
+  const activities = trans(TranslationKeys.PRIMARY_ACTIVITIES, { returnObjects: true });
+  const entry = Object.entries(activities).find((obj) => obj[1] === selectedActivity);
```

Comment:
Just curious, why `Object.entries` instead of `Object.values`? You are never using the key.

Reply:
Fixed